packages/terra-i18n/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Implement locale fallback pattern in translations and locale loaders. Fallback pattern of try regional locale then base locale then en lastly throw error if none provided.
```

Comment:
Maybe rephrase the second sentence to:

Fallback pattern will be first try the regional locale, then try the base locale, then try 'en', and lastly throw an error if en is not provided.